syntax/index.html

```diff
@@ -649,14 +651,20 @@
             Split the header and data rows into cells using the <a>delimiter</a>. Values that are enclosed within the <a>quote character</a> may contain the <a>delimiter</a>. The <a>quote character</a> may be escaped using the <a>escape character</a> where it appears within cells. If the <a>escape character</a> is not the same as the <a>quote character</a> then the <a>escape character</a> is also used to escape the character that immediately follows it.
           </p>
           <p>
-            If <a>trim</a> is <code>true</code> or <code>start</code> then whitespace from the start of values that are not enclosed must be removed from the value. If <a>trim</a> is <code>true</code> or <code>end</code> then whitespace from the end of values that are not enclosed must be removed from the value.
+            If <a>trim</a> is <code>true</code> or <code>start</code> then whitespace from the start of values that are not enclosed MUST be removed from the value. If <a>trim</a> is <code>true</code> or <code>end</code> then whitespace from the end of values that are not enclosed MUST be removed from the value.
           </p>
         </li>
+        <li>
+          Construct <a>embedded metadata</a> as Table Metadata (see <cite><a href="http://www.w3.org/TR/tabular-metadata/#tables"></a></cite> in [[!tabular-metadata]]) using the <code>@context</code> from <a>merged metadata</a> and setting the <code>language</code> annotation to the default language from the <code>@context</code>.</li>
```

Comment:
I made this change as a result of processing the CSV and deriving embedded metadata. If the asserted metadata has `@language`: `en`, and the column titles are then interpreted to be in English, it won't match with the actual column headings from the CSV, unless they are also in English, thus the addition of adding both `@language`: `en` and `language`: `en` to the resulting embedded metadata.

Alternatively, the rules for matching on a title could be relaxed to not consider language, but this could cause false matches for languages that have words with different meaning but the same spelling.

This could be solved by including `'und': 'title'` in asserted metadata, but the examples don't do this, and it seems excessive. Also, if we're asserting titles in English, it does make sense that the CSV contents are also likely in english. It seems we need some way to infer the language of the CSV from metadata. This should probably be discussed in a meeting.